system/src/system_task.cpp

```diff
@@ -785,6 +786,16 @@ void* system_internal(int item, void* reserved)
     case 3: {
         return reinterpret_cast<void*>(system_cloud_get_socket_handle());
     }
+    case 4: {
+        particle::NetIfTester::instance()->testInterfaces();
+        return nullptr;
+    }
+    case 5: {
+        return reinterpret_cast<void*>(&lwip_stats);
+    }
+    case 6: {
+        return (void*)particle::NetIfTester::instance()->getDiagnostics();
+    }
```

Comment:
Helper functions to allow user space app to call into dvos to test the new stuff